requirements.txt

```diff
@@ -1,16 +1,10 @@
 GitPython==2.0.6
 OWSLib
-PyYAML
-cf_units
-dotenv
```

Comment:
I need this! lib/python/aims_realtime_util.py

Reply:
Oops.. I did some grepping of all python code to find what packages were being imported, but somehow I missed that one. I'll add it back in.